system/vi/vi.c

```diff
@@ -4501,6 +4504,9 @@ static void vi_parsecolon(FAR struct vi_s *vi)
     {
       /* Yes... free resources and exit */
 
+      term.c_lflag |= ECHO;
```

Comment:
let's move vi related change to new patch.

Reply:
Got it! I'll add another patch for it then.